defaultPnorLayoutSingleSide.xml

```diff
@@ -174,6 +174,8 @@ Layout Description
         <physicalOffset>0x961000</physicalOffset>
         <physicalRegionSize>0x100000</physicalRegionSize>
         <side>A</side>
+        <xz/>
+        <xzSize>0x100000</xzSize>
```

Comment:
I'm surprised to see that:

1. We hard code the uncompressed size rather than determine it from the original file.
2. We set it to the same value as the physicalRegionSize.  Don't we want to allow 2x or more compression?